doc/integrator/ngeo.rst.tmpl

```diff
@@ -77,11 +82,13 @@ In the controller you have some lines like:
 
 .. code:: javascript
 
-   import gmf<Component>Component from 'gmf/<component>/component.js';
+   import gmf<Component>Module from 'gmf/<component>/module';
+
+This is needed to include the javascript of the used component.
 
-This is needed to include the javascript of the used directives.
+You can add your own imports.
 
-The map configuration will be in the vars file.
+Most of the configuration is in the vars file.
```

Comment:
Can we be more specific?

Reply:
```suggestion
Most of the components configuration and the map configuration is in the ``interfaces_config`` section
of the vars file. See the section below.
```

Reply:
OK thanks.
Small typo: "Most of the component configuration"
Or "Most of the component's configuration"

Reply:
updated thanks